corehq/apps/app_manager/views/utils.py

```diff
@@ -149,6 +149,13 @@ def get_blank_form_xml(form_name):
         'name': form_name,
     })
 
+def get_default_followup_form_xml(form_name, context):
+    context.update({
+        'xmlns': str(uuid.uuid4()).upper(),
+        'name': form_name}
```

Comment:
nitpick: since ```({``` is on the same line, I would put the closing brace and parenthesis on the same line, too.